feature-libs/pickup-in-store/components/container/set-preferred-store/set-preferred-store.component.html

```diff
@@ -20,6 +20,13 @@
     data-text="setPreferredStore.myStore"
     [attr.data-preferred-store]="pointOfServiceName.name"
     class="set-preferred-heading"
+    [attr.aria-label]="
```

Comment:
Maybe we can move this to the component? you can have a method e.g. `getAriaLabel(storeSelected) {...}` and call it from the template: `getAriaLabel((storeSelected$ | async)?.name)`

Reply:
Although looking at the component again I see there are multiple `(storeSelected$ | async)?.name` which creates redundant subscriptions. My suggestion is to add `*ngIf="{ storeSelected: storeSelected$ | async } as data"` to some top-level div or just create new ng-container. This expression will always return true but you will have only one subscription and then everywhere instead of `(storeSelected$ | async)?.name` you can use `data.storeSelected?.name`